site/features.md

```diff
@@ -108,6 +109,19 @@ IP addresses of external services the hosts or containers need to
 connect to. The same IP range must be used everywhere, and the
 individual IP addresses must, of course, be unique.
 
+### <a name="naming-and-discovery"></a>Naming and discovery
+
+Application containers automatically register themselves for discovery
+simply by giving them a container name when they are launched:
+
+    host1$ docker run -dti --name=service ubuntu
+    host1$ docker run -ti ubuntu
+    root@7b21498fb103:/# ping service
+
+This feature supports hot swapping, load balancing and fault
+resilience; see the [weaveDNS](weavedns.html) documentation for more
```

Comment:
"...supports load balancing, fault resilience and hot swapping" (same order as in weaveDNS docs)